discord/interpreter.js

```diff
@@ -174,14 +181,26 @@ module.exports.handleCom = async (message, client) => {
             Logger.log("info", "Received register request")
             var params = command.join(' ').split(',');
             var realName = params[0];
-            var identification = "";
-            if (params.length > 1) {
-                identification = params[1];
-            }
             if (realName === "") {
-                await message.channel.send("Please provide a real name");
+                await message.channel.send("Please provide a real name.");
                 return;
             }
+            var fresher = params[1];
+            if (fresher == null) {
+                fresher = "no";
+            }
+            else {
+                fresher = fresher.trim().toLowerCase();
+            }
+            if (["yes", "\"yes\"", "no", "\"no\""].indexOf(fresher) === -1) {
+                await message.channel.send(`${fresher} is not a valid second argument. Please enter "yes" if you are a fresher, "no" otherwise.`);
+                return;
+            }
+            fresher = fresher === `yes` || fresher === `"yes"`;
```

Comment:
You are switching from string to boolean here which is a bit confusing and not very safe.